docs/initial.md

```diff
@@ -0,0 +1,31 @@
+# OCFL Community Extension Initial
+
+  * **Extension Name:** initial
+  * **Authors:** N Jefferies
+  * **Minimum OCFL Version:** 1.0
+  * **OCFL Community Extensions Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+*Note: This is a placeholder for the special extension name "initial".*
+
+## Overview
+
+This extension definition ensures that the special extension name _initial_ is valid registered extension name.
```

Comment:
Grammar: ".. _initial_ is valid registered.." -> ".. _initial_ is [a] valid registered.."